src/helpers.ts

```diff
@@ -0,0 +1,64 @@
+import fs from 'fs';
+
+export function getFileContent(file: Buffer | string): string {
+  const buffer = Buffer.isBuffer(file) ? file : fs.readFileSync(file);
+
+  return buffer.toString();
+}
+
+/**
+ * Apply a transform to the keys of an object, leaving the values unaffected.
+ */
+export function mapKeys<T>(
```

Comment:
Just use the lodash version? https://lodash.com/docs/4.17.11#mapKeys

Reply:
Okay, I will use lodash

Reply:
Done